❌The linter check is broken

> run golangci-lint
  Running [/home/runner/golangci-lint-1.29.0-linux-amd64/golangci-lint run --out-format=github-actions --enable gofmt,dupl,goconst,goconst,gocyclo] in [] ...
  level=warning msg="[runner] Can't run linter goanalysis_metalinter: S1035: failed prerequisites: [(inspect@github.com/hochfrequenz/go-bo4e/enum/abweichungsgrund, isgenerated@github.com/hochfrequenz/go-bo4e/enum/abweichungsgrund): analysis skipped: errors in package: [/home/runner/work/go-bo4e/go-bo4e/enum/abweichungsgrund/abweichungsgrund_jsonenums.go:6:2: could not import encoding/json (/opt/hostedtoolcache/go/1.18.0/x64/src/encoding/json/decode.go:11:2: could not import encoding (-: could not load export data: cannot import \"encoding\" (unknown iexport format version 2), export data is newer version - update tool)) /home/runner/work/go-bo4e/go-bo4e/enum/abweichungsgrund/abweichungsgrund_jsonenums.go:7:2: could not import fmt (/opt/hostedtoolcache/go/1.18.0/x64/src/fmt/errors.go:7:8: could not import errors (/opt/hostedtoolcache/go/1.18.0/x64/src/errors/wrap.go:8:2: could not import internal/reflectlite (/opt/hostedtoolcache/go/1.18.0/x64/src/internal/reflectlite/swapper.go:8:2: could not import internal/goarch (-: could not load export data: cannot import \"internal/goarch\" (unknown iexport format version 2), export data is newer version - update tool)))) /home/runner/work/go-bo4e/go-bo4e/enum/abweichungsgrund/abweichungsgrund_string.go:5:8: could not import strconv (/opt/hostedtoolcache/go/1.18.0/x64/src/strconv/atof.go:13:8: could not import math (/opt/hostedtoolcache/go/1.18.0/x64/src/math/exp_amd64.go:9:8: could not import internal/cpu (-: could not load export data: cannot import \"internal/cpu\" (unknown iexport format version 2), export data is newer version - update tool)))]]"
  level=warning msg="[runner] Can't run linter unused: buildir: failed to load package goarch: could not load export data: cannot import \"internal/goarch\" (unknown iexport format version 2), export data is newer version - update tool"
  level=error msg="Running error: buildir: failed to load package goarch: could not load export data: cannot import \"internal/goarch\" (unknown iexport format version 2), export data is newer version - update tool"

--- a/.github/workflows/golangci-lint.yml
+++ b/.github/workflows/golangci-lint.yml
@@ -9,7 +9,7 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: golangci-lint
-        uses: golangci/golangci-lint-action@v2
+        uses: golangci/golangci-lint-action@v3
         with:
           version: v1.29
 

